src/app/forms/templates/test-records/section-templates/test/test-section-group1.template.ts

```diff
@@ -0,0 +1,132 @@
+import { FormNode, FormNodeEditTypes, FormNodeTypes, FormNodeViewTypes } from '@forms/services/dynamic-form.types';
+
+export const TestSectionGroup1: FormNode = {
```

Comment:
Why is this called `TestSectionGroup1`? I do not see any group 2 or 3 🙂

Reply:
It's to do with the taxonomy that we know we're going to have to handle in the future, we'll probably have to handle around 16 groups, this is just the first step, it'll have to be refactored later